pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -700,6 +703,56 @@ var _ = Describe("ImageUpload", func() {
 			validatePVCDefaultInstancetypeLabels()
 		})
 
+		It("Should create DataSource pointing to the PVC", func() {
```

Comment:
The test could assert that the DV is pointing to the expected PVC and that it has the expected labels.

Reply:
Do you mean to DS?

Reply:
Done! Thanks :)